test/Microsoft.AspNetCore.Mvc.Core.Test/Internal/DisableRequestSizeLimitResourceFilterTest.cs

```diff
@@ -0,0 +1,154 @@
+            var disableRequestSizeLimitResourceFilterFinal = new DisableRequestSizeLimitResourceFilter(NullLoggerFactory.Instance);
+            var resourceExecutingContext = CreateResourceExecutingContext(new IFilterMetadata[] { disableRequestSizeLimitResourceFilter, disableRequestSizeLimitResourceFilterFinal });
+
+            var httpMaxRequestBodySize = new TestHttpMaxRequestBodySizeFeature();
+            resourceExecutingContext.HttpContext.Features.Set<IHttpMaxRequestBodySizeFeature>(httpMaxRequestBodySize);
+
+            // Act
+            disableRequestSizeLimitResourceFilter.OnResourceExecuting(resourceExecutingContext);
+            disableRequestSizeLimitResourceFilterFinal.OnResourceExecuting(resourceExecutingContext);
+
+            // Assert
+            Assert.Null(httpMaxRequestBodySize.MaxRequestBodySize);
+            Assert.Equal(1, httpMaxRequestBodySize.Count);
+        }
+
+        [Fact]
+        public void LogsFeatureNotFound()
+        {
+            // Arrange
+            var sink = new TestSink();
+            var loggerFactory = new TestLoggerFactory(sink, enabled: true);
+
+            var disableRequestSizeLimitResourceFilter = new DisableRequestSizeLimitResourceFilter(loggerFactory);
+            var resourceExecutingContext = CreateResourceExecutingContext(new IFilterMetadata[] { disableRequestSizeLimitResourceFilter });
+
+            // Act
+            disableRequestSizeLimitResourceFilter.OnResourceExecuting(resourceExecutingContext);
+
+            // Assert
+            var write = Assert.Single(sink.Writes);
+            Assert.Equal($"A request body size limit could not be applied. This server does not support the IHttpRequestBodySizeFeature.",
+                write.State.ToString());
+        }
+
+        [Fact]
+        public void LogsFeatureIsReadOnly()
+        {
+            // Arrange
+            var sink = new TestSink();
+            var loggerFactory = new TestLoggerFactory(sink, enabled: true);
+
+            var disableRequestSizeLimitResourceFilter = new DisableRequestSizeLimitResourceFilter(loggerFactory);
+            var resourceExecutingContext = CreateResourceExecutingContext(new IFilterMetadata[] { disableRequestSizeLimitResourceFilter });
+
+            var httpMaxRequestBodySize = new TestHttpMaxRequestBodySizeFeature();
+            httpMaxRequestBodySize.IsReadOnly = true;
+            resourceExecutingContext.HttpContext.Features.Set<IHttpMaxRequestBodySizeFeature>(httpMaxRequestBodySize);
+
+            // Act
+            disableRequestSizeLimitResourceFilter.OnResourceExecuting(resourceExecutingContext);
+
+            // Assert
+            var write = Assert.Single(sink.Writes);
+            Assert.Equal($"A request body size limit could not be applied. The IHttpRequestBodySizeFeature for the server is read-only.", write.State.ToString());
+        }
+
+        [Fact]
+        public void DisableRequestSizeLimitAttribute_LogsMaxRequestBodySizeSetToNull()
```

Comment:
Has nothing to do w/ `DisableRequestSizeLimitAttribute`; just call this `LogsMaxRequestBodySizeSetToNull`